src/front/glsl_new/error.rs

```diff
@@ -0,0 +1,44 @@
+use std::fmt;
+use std::io;
+
+#[derive(Debug)]
+pub enum ErrorKind {
+    InvalidInput,
+    IoError { error: io::Error },
```

Comment:
nit: any reason to have the named field here? seems very straightforward to just have `IoError(io::Error)`

Reply:
Done